src/assets/YAML/default/CultureAndOrganization/EducationAndGuidance.yaml

```diff
@@ -24,11 +24,30 @@ Culture and Organization:
       isImplemented: false
       evidence: ""
       comments: ""
+    Security Coaching:
+      risk: Even if security practices are understood, it doesn't mean that they get implemented.
```

Comment:
Suggestion: Training does not change behaviour. Therefore, even if security practices are understood, it's likly that they are not performed.